spec/features/locations_request_spec.rb

```diff
@@ -43,19 +42,25 @@
       fill_in 'Notes', with: 'cool notes'
 
       click_button "Update Location"
-
       location.reload
 
       location.contact_info.should == 'someone'
       location.notes.should == 'cool notes'
     end
+
+    it "can archive a location that is no longer available" do
+      visit edit_location_path(location)
+      click_button "Archive Location"
+      location.reload
```

Comment:
right! I think that I was trying something else and forgot to take that out. Good catch.